AIDevGallery/Samples/SharedCode/GenAIModel.cs

```diff
@@ -19,12 +19,12 @@ internal class GenAIModel : IChatClient, IDisposable
     private const string TEMPLATE_PLACEHOLDER = "{{CONTENT}}";
 
     // Search Options
-    private const int DefaultTopK = 50;
-    private const float DefaultTopP = 0.9f;
-    private const float DefaultTemperature = 1;
-    private const int DefaultMinLength = 0;
+    public const int DefaultTopK = 50;
```

Comment:
We should move away from having these as public, not making them all public. We should not rely on GenAIModel class for these values. Ideally, these are provided to the sample, or recovered in a generic way, since different implementations of IChatClient will have different defaults, and we want the sample to work with all of them.